transcom/mymove
# B 21557 estimated price

## [B-21557](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240)
## [INT PR](https://github.com/transcom/mymove/pull/14448)
## [2nd INT PR](https://github.com/transcom/mymove/pull/14579)
## Summary

### How to test

1. Create a move with an NTS-R, or find an existing move that is already approved
2. As a SC add an NTS-R shipment to the move and submit the move to SC.
3. As a TOO, go to the move task order tab, and edit the NTS-R to add a previously recorded weight then save
4. The service items should have prices associated with them.
5. Edit the NTS-R shipment again to modify the "previously recorded weight", can be any nonzero value, then save
6. The values should be updated based on the new estimated weight
